nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/AbstractCommentManager.java

```diff
@@ -193,4 +205,80 @@ protected void notifyEvent(CoreSession session, String eventType, DocumentModel
         notifyEvent(session, eventType, commentParent, commentDocumentModel);
     }
 
+    /**
+     * Checks if a document has comments.
+     *
+     * @param session the core session
+     * @param document the document model who's comments are being counted
+     * @return true if comments were found
+     * @since 11.1
+     */
+    protected boolean hasComments(CoreSession session, DocumentModel document) {
+        // CommentManagerImpl doesn't support it.
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Checks if a document has comments from a particular user.
+     *
+     * @param session the core session
+     * @param document the document model who's comments are being counted
+     * @return true if comments were found
```

Comment:
`true if comments were found` -> `{@code true}if comments were found, otherwise {@code false}`
wdyt

Reply:
Well, the otherwise part is implicit 😄

Reply:
Also corrected the javadoc for `#hasCommentsByUser`which was missing details and param